Habit/Habit/MainScreen/MainScreenPresenter.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  MainScreenPresenter.swift
+//  Habit
+//
+//  Created by Мария on 07.04.2020.
+//  Copyright © 2020 Мария. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol MainScreenPresenterProtocol: class {
+    
+    var router: MainScreenRouterProtocol! {set get }
+    func returnNumberOfSections() -> Int
```

Comment:
лучше делать computed property, если тебе надо что-то вернуть и нет входных параметров, а сложность выполнения не превышает O(n) (знаешь что это?)
```suggestion
    var numberOfSections: Int { get }
```